corehq/preindex/couchviews.lock

```diff
@@ -97,6 +97,8 @@ ae04d8c82218a9543491 corehq/couchapps/registry_data_sources/views/view/reduce.js
 dc481a24b6799cc81f9d corehq/couchapps/registry_data_sources_by_last_modified/views/view/map.js
 b30cf4ab3c1f92248da7 corehq/couchapps/registry_report_configs/views/view/map.js
 ae04d8c82218a9543491 corehq/couchapps/registry_report_configs/views/view/reduce.js
+bd020a70a1ba1fe7432c corehq/couchapps/repeat_records_by_last_checked/views/view/map.js
+ae04d8c82218a9543491 corehq/couchapps/repeat_records_by_last_checked/views/view/reduce.js
```

Comment:
Hey @millerdev  Even though I have updated `couchviews.lock` and [added](https://github.com/dimagi/commcare-hq/pull/32726/files#diff-03314a42b69c7ccde53ec4ed16f62c6b14e7dc2a792b3b6442eaa45ea01a9688R679) a migration, the migration test still expects another migration file and [fails](https://github.com/dimagi/commcare-hq/actions/runs/4402456300/jobs/7709638510#step:5:7205). Do you know what I am missing here?

(This is just a draft PR just to push whatever test commits,  the actual PR is https://github.com/dimagi/commcare-hq/pull/32626 where I will push the final clean commit)

Reply:
It appears that you have changed view files since you last ran `makemigrations`. Steps to resolve:

```sh
# delete preindex 0002 migration since it's unnecessary to create a second one exactly like it
rm corehq/preindex/migrations/0002_reindex_repeat_records.py

# re-run makemigrations
./manage.py makemigrations preindex -n reindex_repeat_records
```

Commit and push.

Reply:
Thanks! Just this morning when I woke up, I realized the same!